### PR TITLE
Moving deprecations along.

### PR DESCRIPTION
**Pull request by jmdavis**
This is mostly just putting dates on existing deprecations that were
missing dates, but it does remove a few things that have been deprecated
long enough to be removed.

### CODE REVIEW

**Comment by schveiguy** [comment]
LGTM

File: std/stdio.d [hunk 1]
```diff
@@ -2466,12 +2466,6 @@ $(D Range) that locks the file and allows fast writing to it.
         _iobuf* handle_;     // the unshared version of fps
         int orientation_;
     public:
-        deprecated("accessing fps/handle/orientation directly can break LockingTextWriter integrity")
```
Comment (burner) [inline std/stdio.d]: git blame says this has been deprecated for ~1.5Years now, but no remove date was ever mentioned.

please do not get me wrong. This should go, but IMO we first sould say when.

File: std/syserror.d [hunk 2]
```diff
@@ -1,52 +0,0 @@
-// Written in the D programming language
```
Comment (burner) [inline std/syserror.d]: see my comment for std.stdio

File: std/windows/registry.d [hunk 3]
```diff
@@ -1292,12 +1292,6 @@ public:
         return value;
     }
 
-    deprecated("Please use value_QWORD instead.")
```
Comment (burner) [inline std/windows/registry.d]: see my comment for std.stdio

File: std/stdio.d (hunk 1 above)
Comment (jmdavis) [inline std/stdio.d]: The policy is about 1 year as deprecated and documented, and 1 year as deprecated and undocumented. This was undocumented back in 2013 even before it was deprecated. Yes, we would ideally put a note in the documentation about when it's going to go away, but even then, the date that we normally give for removal is actually the date that it's removed from the docs, not from the code, and if something has been deprecated for ~1.5 years and undocumented for longer than that, then no one should be using it by this point anyway. They've been given more than ample warning.

**Comment by ZombineDev** [comment]
LGTM

**Comment by quickfur** [comment]
LGTM

**Comment by quickfur** [comment]
Auto-merge toggled on

**Pull request merged**